app/views/privacy/privacy-policy.njk

```diff
@@ -78,6 +79,19 @@
     <p class='mpdp-text-color-dark-grey govuk-!-margin-top-8 govuk-!-margin-bottom-0'>This notice was last updated on 22 September 2022. </p>
 
   </div>
+  <div class="govuk-grid-column-one-third">
+    {{relatedContent([
+      {
```

Comment:
You have modified the relatedContent perfectly. I would suggest we put the links (array being passed) in a config file so that we can reuse them if needed and pass them from the model of the actual page.
For example, within app/config create another file called relatedContent that will hold a list of all the links with an exposed get method so that we can get links which are linked to specific page. The benefit of doing so is that we if we need to add existing links to other page we can just modify it one place. Again for example the list could like:
`[{
        id: "ffTCLink",
        text: "Terms and conditions",
        url: "https://www.gov.uk/help/terms-conditions",
        pages: ['privacy-policy']
   }]`

Hope that makes sense but give me shout if you want to discuss further.

Reply:
Thanks for your comment,  I've updated my code.